clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -432,4 +432,19 @@ DPCT_NON_ENUM_OPTION(
         "Query functionally compatible SYCL API to migrate CUDA API."),
     llvm::cl::value_desc("api"), llvm::cl::cat(DPCTCat), llvm::cl::Optional)
 
+DPCT_ENUM_OPTION(
+    DPCT_OPT_TYPE(static llvm::cl::bits<HelperFuncPreference>), Preferences,
+    "helper-function-preference",
+    DPCT_OPTION_VALUES(DPCT_OPT_ENUM(
+        "no-queue-device", int(HelperFuncPreference::NoQueueDevice),
+        "Call SYCL API to get queue and device instead of calling helper function. Notice\n"
```

Comment:
Call SYCL API to get queue and device instead of calling helper function.

Reply:
fixed